src/core/interpreter.h

```diff
@@ -32,7 +32,21 @@ class ObjectExplorer {
 
 class Interpreter {
  public:
-  using RedisFunc = std::function<void(MutSliceSpan, ObjectExplorer*)>;
+  // Arguments received from redis.call
+  struct CallArgs {
+    // Full arguments, including cmd name.
+    MutSliceSpan args;
+
+    // Backing storage for args (excluding cmd name).
+    // Can be moved out to efficiently store for async dispatch.
+    std::string&& buffer;
```

Comment:
The arguments point to the string that is placed on the stack of a function that prepares this call. We cannot _move out of_ the string if we want to read from it, because moving it under sso will invalidate the pointers.

Reply:
I find it difficult to comprehend why a member is rvalue. Maybe it's just me. I do not understand why you introduce it and never access.